### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by henkmollema**
- ASP.NET 5.0 -> ASP.NET 5
- KVM -> DNVM
- Minor text changes

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -33,7 +35,7 @@ To deploy MusicStore on Heroku, click the button below:
 [![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy)
 
 ###NTLM authentication
-More information at src/MusicStore/StartupNtlmAuthentication.cs.
+More information at [src/MusicStore/StartupNtlmAuthentication.cs](https://github.com/aspnet/MusicStore/blob/dev/src/MusicStore/StartupNtlmAuthentication.cs).
```
Comment (Eilon) [inline README.md]: Can you make the links just be `[src/whatever.cs]` so that it'll work in any branch?

File: README.md [hunk 2]
```diff
@@ -21,11 +22,12 @@ This project is part of ASP.NET 5.0. You can find samples, documentation and get
 	6. Run `dnx . run` (This hosts the app in a console application - Application started at URL **http://localhost:5003/**).
 
 ## To run the sample on Mac/Mono:
-* Follow [Home](https://github.com/aspnet/home) instructions to install mono, kvm on Mac.
-* Open a command prompt and cd `\src\MusicStore\`.
+* Follow the instructions at the [Home](https://github.com/aspnet/Home) repository to install Mono and DNVM on Mac OSX.
```
Comment (Eilon) [inline README.md]: OSX -> OS X

**Comment by henkmollema** [comment]
@Eilon updated and squashed.

**Comment by henkmollema** [comment]
By the way, this PR targets the `dev` branch which is not the default branch yet.

**Pull request merged**